contracts/payloads/IGP95/PayloadIGP95.sol

```diff
@@ -0,0 +1,876 @@
+                supplyTokens[1] = ETH_ADDRESS;
+
+                address[] memory borrowTokens = new address[](0);
+
+                // Pause the user operations
+                LIQUIDITY.pauseUser(FLUID_ETH_DEX, supplyTokens, borrowTokens);
+            }
+        }
+    }
+
+    // @notice Action 8: Set Launch Limits for RLP-USDC DEX
+    function action8() internal isActionSkippable(8) {
+        address RLP_USDC_DEX = getDexAddress(28);
+        {
+            // RLP-USDC DEX
+            DexConfig memory DEX_RLP_USDC = DexConfig({
+                dex: RLP_USDC_DEX,
+                tokenA: RLP_ADDRESS,
+                tokenB: USDC_ADDRESS,
+                smartCollateral: true,
+                smartDebt: false,
+                baseWithdrawalLimitInUSD: 6_500_000, // $6.5M
+                baseBorrowLimitInUSD: 0, // $0
+                maxBorrowLimitInUSD: 0 // $0
+            });
+            setDexLimits(DEX_RLP_USDC); // Smart Collateral
+
+            DEX_FACTORY.setDexAuth(RLP_USDC_DEX, TEAM_MULTISIG, false);
+        }
+
+        {
+            address fSL28_RLP_USDC = getSmartLendingAddress(28);
+
+            // set rebalancer at fSL28 to reserve contract proxy
+            ISmartLendingAdmin(fSL28_RLP_USDC).setRebalancer(
+                address(FLUID_RESERVE)
+            );
+        }
+        {
+            IFluidDex(RLP_USDC_DEX).updateMaxSupplyShares(
+                5_000_000 * 1e18 // $5M
+            );
+        }
+    }
+
+    // @notice Action 9: Set Launch Limits for Gold Vaults and DEX
+    function action9() internal isActionSkippable(9) {
+        {
+            // PAXG-XAUT DEX
+            address PAXG_XAUT_DEX = getDexAddress(32);
+            {
+                DexConfig memory DEX_PAXG_XAUT = DexConfig({
+                    dex: PAXG_XAUT_DEX,
+                    tokenA: PAXG_ADDRESS,
+                    tokenB: XAUT_ADDRESS,
+                    smartCollateral: true,
+                    smartDebt: false,
+                    baseWithdrawalLimitInUSD: 1_000_000, // $1M
+                    baseBorrowLimitInUSD: 0, // $0
+                    maxBorrowLimitInUSD: 0 // $0
+                });
+                setDexLimits(DEX_PAXG_XAUT); // Smart Collateral
+
+                DEX_FACTORY.setDexAuth(PAXG_XAUT_DEX, TEAM_MULTISIG, false);
+            }
+            {
+                IFluidDex(PAXG_XAUT_DEX).updateMaxSupplyShares(
+                    1_800_000 * 1e18 // $1.8M
+                );
+            }
+        }
+
+        {
+            // [TYPE 1] XAUT / USDC VAULT
+            address XAUT_USDC_VAULT = getVaultAddress(116);
+            {
+                VaultConfig memory VAULT_XAUT_USDC = VaultConfig({
+                    vault: XAUT_USDC_VAULT,
+                    vaultType: VAULT_TYPE.TYPE_1,
+                    supplyToken: XAUT_ADDRESS,
+                    borrowToken: USDC_ADDRESS,
+                    baseWithdrawalLimitInUSD: 750_000, // $750k
```

Comment:
all the base limits here, let's get them to same as maxBorrowLimit. Can just set the same value 

for base withdrawal limit, where <1M, let's set 1M$

Reply:
i didnt get this?